client/landing/jetpack-cloud/sections/pricing/jpcom-footer/style.scss

```diff
@@ -0,0 +1,128 @@
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
+
+.jpcom-footer {
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+
+	padding: 12px 16px 24px;
+
+	@include break-mobile {
+		padding: 12px 12px 24px;
+	}
+
+	@include breakpoint-deprecated( '>660px' ) {
+		padding: 40px 0 24px;
+	}
+}
+
+.jpcom-footer__separator {
+	// This color comes from design (not present in color-studio).
+	background: #c9d7e1;
+
+	margin: 0;
+
+	// These 3 properties make the line go across the whole viewport even when the
+	// parent component doesn't go that far.
+	position: relative;
+	left: calc( -50vw + 50% );
+	width: 100vw;
+}
+
+.jpcom-footer__col {
+	flex: 0 0 100%;
+
+	padding: 10px 0;
+
+	@include break-mobile {
+		flex: 0 0 calc( 50% - 24px );
+
+		padding: 10px 12px;
+	}
+
+	// We need this deprecated breakpoint since Calypso still uses it to determine
+	// the layout padding.
+	@include breakpoint-deprecated( '>660px' ) {
+		flex: 0 0 calc( 50% - 12px );
+
+		padding: 10px 0;
+	}
+
+	@include break-large {
+		flex: 0 0 25%;
+	}
+
+	@include break-xlarge {
+		flex: 0 0 20%;
+	}
+}
+
+.jpcom-footer__sub-col {
+	margin-top: 20px;
+}
+
+.jpcom-footer__links {
+	list-style: none;
+	list-style-image: none;
+
+	margin: 0;
+}
+
+.jpcom-footer__link {
+	padding-bottom: 8px;
+}
+
+.jpcom-footer__col-header {
+	font-weight: bold;
```

Comment:
I believe the convention is to use numbers, in this case 700.

Reply:
Good catch Kev!